tests/test_shell_pr_probe_reftable_and_multi_remote.py

```
@@ -0,0 +1,235 @@
+        repo = base / shell / "slug" / name / "repo"
+        (repo / ".git").mkdir(parents=True, exist_ok=True)
+        (repo / ".git" / "HEAD").write_text("ref: refs/heads/main\n", encoding="utf-8")
+        (repo / ".git" / "config").write_text(config, encoding="utf-8")
+        rc, out, err = _run(shell, shell_args, script, f'_cmux_github_repo_slug_for_path "{repo}"', {})
+        if rc != 0 or out.strip() != expected:
+            failures.append(f"{shell}: slug {name} expected {expected}, got rc={rc} out={out!r} err={err!r}")
+    return failures
+
+
+def main() -> int:
+    base = Path("/tmp") / f"cmux_pr_probe_reftable_multi_remote_{os.getpid()}"
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)